### PR TITLE
Fix hanging spinners when editing both location and pickup date

### PR DESCRIPTION
**Pull request by nicktiberi**
I have no idea why changing the timing of the uncheck logic makes any difference, but it does.

### CODE REVIEW

**Comment by awead** [comment]
My guess is Javascript's sometimes unintelligible async processing.

**Review by banukutlu — COMMENTED** [review]
fixes the hanging spinner 👍🏻 
CI still failing though :/

File: spec/features/holds_spec.rb [hunk 1]
```diff
@@ -39,11 +39,11 @@
 
       context 'when the user successfully changes the pickup library of the same hold more than once' do
         it 'success badges gets cleared each time', js: true do
-          page.check 'hold_list__3911148'
+          page.find("input[type='checkbox'][id='hold_list__3911148']").click
           page.select 'Berks Campus Library', from: 'pickup_library'
           page.click_button 'Update Selected Holds'
 
-          expect(page).to have_css '.badge-success', text: 'Successfully changed pickup location', count: 1
+          expect(page.find('.badge-success')).to have_text 'Successfully changed pickup location', count: 1
```
Comment (nicktiberi) [inline spec/features/holds_spec.rb:46]: @banukutlu LGTM! ✅

**Review by banukutlu — APPROVED** [review]
@nicktiberi thanks for the fix! lgtm 👍🏻

**Review by whereismyjetpack — APPROVED** [review]

**Pull request merged**